features/order/managing_orders/cancelling_order_with_payment.feature

```diff
@@ -0,0 +1,22 @@
+@managing_orders
+Feature: Checking payment state of a cancelled order
+    In order to check payment state after cancelling an order
+    As an Administrator
+    I want to payment has proper state
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And there is a customer "john.doe@gmail.com" that placed an order "#00000022"
+        And the customer bought a single "PHP T-Shirt"
+        And the customer chose "Free" shipping method to "France" with "Cash on Delivery" payment
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Checking payment state of a placed order
+        When I view the summary of the order "#00000022"
+        And I cancel this order
+        Then I should be notified that it has been successfully updated
+        Then it should have cancelled payment state
```

Comment:
How about adding this single assertion to already existing [cancel order feature](https://github.com/michalmarcinkowski/Sylius/blob/5c266bb1602a16f0862b7ffce373b14fc71d5f65/features/order/managing_orders/cancelling_order.feature#L23-L23)?

We could check all 3 states (order state, order payment state and order shipment state) in one scenario and avoid duplicated scenarios.